src/controllers/issuer.ts

```diff
@@ -503,6 +503,67 @@ export class IssuerController {
 		}
 	}
 
+	/**
+	 * @openapi
+	 * 
+	 * /resource/search/{did}/{resourceId}:
+	 *   get:
+	 *     tags: [ Resource ]
+	 *     summary: Get a DID-Linked Resource.
+	 *     description: This endpoint returns the DID-Linked Resource for a given DID identifier and resource identifier.
+	 *     parameters:
+	 *       - in: path
+	 *         name: did
+	 *         description: DID identifier
+	 *         schema:
+	 *           type: string
+	 *         required: true
+	 *         example: did:cheqd:mainnet:7bf81a20-633c-4cc7-bc4a-5a45801005e0
+	 *       - in: path
+	 *         name: resource Identifier
+	 *         description: Fetch a DID-Linked Resource by Resource ID unique identifier.
+	 *         schema:
+	 *           type: string
+	 *           format: uuid
+	 *         required: true
+	 *         example: 3ccde6ba-6ba5-56f2-9f4f-8825561a9860
```

Comment:
1. Move `resourceId` to optional path parameters
2. Move other resource-specific query parameters here

Reply:
@ankurdotb 
If I understand correctly, in this case, we have 2 `resourceId:` first one in the path parameter and second one in the query parameter? If so, IMHO it will be good if we only provide the `resourceId` like query parameter so as not to confuse the user. What do you think?

Reply:
Done ✅ 
PS: left `resourceId` only in the `resource/search` query parameters.